Assets/Scripts/Customization/CharacterCustomization.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[ExecuteInEditMode]
+public class CharacterCustomization : MonoBehaviour {
+    public static CharacterCustomization instance;
+
+    private Renderer _renderer;
+
+    [SerializeField]
+    private bool _hasCharacter = false, 
+                 _isNPC = false;
+
+    public bool IsCreatingCharacter = false;

```

Comment:
false is default value for boolean, Don't define default compiler will do this for you.